doc/manual/documentation.rst

```diff
@@ -99,14 +99,17 @@ documentation:
        ...
        """
 
-5. Group examples under an ``# Examples`` section and use `````julia`` blocks instead of
-   standard text.
+5. Group examples under an ``# Examples`` section and use Markdown `Code blocks`_ instead of
```

Comment:
I would mention in the first sentence that writing doctests is recommended. Examples that cannot be checked automatically should only be used when doctests wouldn't work.